src/main/java/io/confluent/connect/jdbc/util/DateTimeUtils.java

```diff
@@ -159,15 +244,28 @@ public static Timestamp toTimestamp(String nanos) throws NumberFormatException {
    * @param tz the timezone of the source database
    * @return the equivalent java sql Timestamp
    */
-  public static Timestamp toTimestampFromIsoDateTime(String isoDT, TimeZone tz) {
+  public static Timestamp toTimestampFromIsoDateMicrosTime(String isoDT, TimeZone tz) {
```

Comment:
nit: Fix the javadoc
```
/**
* Get {@link Timestamp} from epoch with micro precision
*
* @param isoDT iso dateTime format "yyyy-MM-dd'T'HH:mm:ss.SSSSSS"
```

Reply:
We need to have 6 `S` in the format. Looks like there are 3 additional currently.